app/data/models/funding-option.js

```diff
@@ -1,8 +1,7 @@
 module.exports = (sequelize, DataTypes) => {
   const fundingOption = sequelize.define('fundingOption', {
     fundingCode: { type: DataTypes.STRING, primaryKey: true, autoIncrement: true },
```

Comment:
what  mad man made this auto-incr?!

`fundingCode` is like `80001` being Arable soil: Intro

could you remove please?